src/start.sh

```diff
@@ -87,6 +87,10 @@ if [ "${INSTALL_DEV_TOOLS:-0}" -gt 0 ] ; then
     apk add --no-cache nano less
 fi
 
+# Add an alias for padd to the root user's bashrc
```

Comment:
placed in start.sh will do also, but bashrc will be rewritten at each container start, erasing possible user aliases and/or bash user defined settings.

Reply:
It needs to be in `start.sh` because the user-defined `FTLCONF_` environment variables will not be defined at build-time, so it need to be applied at run time.

@yubiuser another thought, to ensure we're not munging any user-defined `.bashrc` (though, one would argue there isn't much of a need to do so in a container such as this) Perhaps we cna do something like:

Dockerfile:
```dockerfile
ADD --chmod=0755 https://raw.githubusercontent.com/pi-hole/PADD/PADD_FTLv6/padd.sh /usr/bin/padd
```

start.sh
```bash
port="${FTLCONF_webserver_port%%,*}"
echo "/usr/bin/padd --port ${port:-8080} --secret ${FTLCONF_webserver_api_password}'" > /usr/local/bin/padd
chmod +x /usr/local/bin/padd
```

Reply:
> using $ prevent bash to substitute variable at buildtime.

Ah OK thanks, when I was tinkering about before it wasn't working - but that could have been for all manner of reasons, to be honest!

Reply:
I've just noticed that the slash before the $ was removed in the reply when publishing the comment, there is may be a misunderstanding : here is a picture of the dockerfile of how it should look:
![image](https://github.com/pi-hole/docker-pi-hole/assets/3601462/bf96b6ec-4151-4d50-8102-60ad3477daf7)

and the result in the image:
![image](https://github.com/pi-hole/docker-pi-hole/assets/3601462/bf683491-260d-4a9a-8975-9ca0567af7ee)